distributed/scheduler.py

```diff
@@ -6304,6 +6305,20 @@ async def register_worker_plugin(self, comm, plugin, name=None):
         )
         return responses
 
+    async def unregister_worker_plugin(self, comm, name):
+        """ Unregisters a worker plugin"""
+
+        for idx, entry in enumerate(self.worker_plugins):
+            if entry["name"] == name:
+                break
+        else:
+            raise KeyError(f"The worker plugin {name} does not exists")
```

Comment:
Nitpick: raising a `ValueError` probably makes more sense here as this is more about an incorrect value (for `name`) rather than for a missing value in a mapping (though this is a very minor point)